### ISSUE TITLE
Questions about calling "multipass" option

### ISSUE THREAD

**Issue by wl879** [svg/svgo#1330]
The source code is in [/lib/svgo.js:41](https://github.com/svg/svgo/blob/master/lib/svgo.js#L41)

```javascript
const optimize = (svgstr, config) => {
  ...
  const maxPassCount = config.multipass ? 10 : 1;
  ...
  for (let i = 0; i < maxPassCount; i += 1) {
    svgjs = svg2js(svgstr);   //  ⚠️ "svg2js" always using origin data "svgstr"
    ～～～～～～～～～～～～～～～～～～～～～～～～～～～～～～～～
   ...
  }
  ...
```

I think, "svgstr" should be the last optimized data.

```javascript
 for (let i = 0; i < maxPassCount; i += 1) {
    svgjs = svg2js(svgstr); 
    ....
    svgjs = js2svg(svgjs, config.js2svg);
    ...
    svgstr = svgjs.data
    ～～～～～～～～～
  }
```

**Comment by TrySound** [comment]
Right, need to add tests

**Comment by TrySound** [comment]
Fixed in 2.0.1
